acceptance/tests/storeconfigs/legacy_storeconfigs_import_export.rb

```diff
@@ -1,75 +0,0 @@
-puppetdb_query_url = "http://localhost:8080/pdb/query"
```

Comment:
Now this test makes sense to remove, I'm not sure anything else in this directory does though ...